src/gatsby-node.js

```diff
@@ -70,6 +70,74 @@ const createWebsocketEvent = async function(
   }
 };
 
+const modifiedWebSocketEvent = async function(
+  data,
+  createNode,
+  getNode,
+  deleteNode,
+  token,
+  baseUrl,
+  expansions,
+  backlinks,
+  searchParams
+) {
+  let urlChild = data['modified'][0]['@id'];
+  let urlParent = data['modified'][0]['parent']['@id'];
+  let urlList = [urlChild, urlParent];
+  for (const url of urlList) {
+    try {
+      for await (const node of ploneNodeGenerator(
+        url,
+        token,
+        baseUrl,
+        expansions,
+        backlinks
+      )) {
+        console.log(`Creating node – ${node.id.replace(baseUrl, '') || '/'}`);
+        createNode(node);
+      }
+    } catch (err) {
+      console.error(`Skipping node – ${url.replace(baseUrl, '')} (${err})`);
+      let node = getNode(url);
+      let breadcrumbsNode = getNode(`${url}/@breadcrumbs`);
+      let navigationNode = getNode(`${url}/@navigation`);
+      if (node) {
+        console.log(`node deleted at ${url}`);
+        deleteNode({ node: node });
+      }
+      if (breadcrumbsNode) {
+        deleteNode({ node: breadcrumbsNode });
+      }
+      if (navigationNode) {
+        deleteNode({ node: navigationNode });
+      }
+    }
+  }
+  await childItemsForAUrl(urlChild, token, baseUrl, createNode, searchParams);
+};
+
+const childItemsForAUrl = async function(
```

Comment:
I don't recall seeing articles in functions, so I believe this should be `childItemsForUrl`instead.